Anagram/Anagram.cs

```diff
@@ -0,0 +1,85 @@
+﻿
+namespace Anagram
+{
+    public partial class Anagram : Form
+    {
+        private String[] wordList;
```

Comment:
self explanatory: word list means any list of words right? and this is not the case.